var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -381,6 +381,11 @@ def cmake_args(self):
                 args.append(self.define('CMAKE_Fortran_COMPILER',
                                         spec['mpi'].mpifc))
 
+        # work-around for https://github.com/HDFGroup/hdf5/issues/1320
+        if spec.satisfies('@1.10.8') and sys.platform != 'windows':
```

Comment:
Looking at the linked issue, should this be `@1.10.8:`?

Reply:
Or now, it's likely 1.10.8, dev and 1.13.0.
I only tested 1.10.8 because I had to stop working in the evening, but we can check if the var is called the same on 1.13.0 still.
No, 1.12 is not affected and I think there is no back port release to the 1.12 series yet that would ship the broken patch

Reply:
```suggestion
        if spec.satisfies('@1.10.8,1.13.0'):
```

Leave the windows support for the windows branch?